### PR TITLE
#3482 [Demisto] release a new plugin

### PR DESCRIPTION
**Pull request by SepioSystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by SepioSystems** [comment]
Release a new plug-in version to fix lost events using eventid instead of timestamp

**Review by ilaner — CHANGES_REQUESTED** [review]
Please pull from origin/master and add release notes

File: Packs/Sepio/ReleaseNotes/SeprioPrimeAPI_FromEventId.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Sepio Prime API Test Playbook with all the integration supported commands
```
Comment (ilaner) [inline Packs/Sepio/ReleaseNotes/SeprioPrimeAPI_FromEventId.md:1]: In this folder supposed to be the release notes file, not the readme. It should be named by the version of the pack (for instance, `1_0_6.md`) and should include a list of changes.
Take an example from the other files in this folder.

**Review by ilaner — CHANGES_REQUESTED** [review]
Please update release notes, update the `pack-metadata.json`

**Pull request merged**